modules/log6x/logging-http-forward-6-2.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/log_collection_forwarding/configuring-log-forwarding.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="logging-http-forward-6-2_{context}"]
+= Forwarding logs over HTTP
+
+To enable forwarding logs over HTTP, specify `http` as the output type in the `ClusterLogForwarder` custom resource (CR).
+
+.Procedure
+
+* Create or edit the `ClusterLogForwarder` CR using the template below:
++
+.Example ClusterLogForwarder CR
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: <log_forwarder_name> <1>
+  namespace: <log_forwarder_namespace> <2>
+spec:
+  managementState: Managed
+  outputs:
+    - http:
+        headers: <3>
+          h1: v1
+          h2: v2
+        method: POST
+        proxyURL: <proxy_url> <4>
+        url: <url>   <5>
+      name: httpout-app
+      type: http
+      tls:
+        insecureSkipVerify: <6>
+        ca:
+          secretName: <secret_name>  <7>
+          key: <secret_key>
+  pipelines:
+    - inputRefs:
+        - application
+      name: <pipeline_name> 
+      outputRefs:
+        - httpout-app <8>
+  serviceAccount:
+    name: <service_account_name> <9>
+----
+<1> In legacy implementations, the CR name must be `instance`. In multi log forwarder implementations, you can use any name.
+<2> In legacy implementations, the CR namespace must be `openshift-logging`. In multi log forwarder implementations, you can use any namespace.
```

Comment:
Do we still need to mention `legacy implementations`? For the API `clusterlogforwarders.observability.openshift.io`, we don't have a legacy implementation.

Reply:
removed